docs/DeveloperGuide.md

```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
 
 Team size: 5
 
-1\. **Allow names to include symbols and special characters**
+1\. **Enhance Name Validation to Support Special Characters and Improve Formatting**
 
-  As of the current version of MediBase3, names are restricted to both alphanumerics and spaces only. 
-  However, there are people with legal names that include symbols and special characters, such as 'Nagaratnam s/o Suppiah'. In the future, we will loosen the restrictions for names to allow users to input names with any special characters and symbols.
+   Currently, MediBase3 restricts patient names to alphanumerics and spaces only, preventing the inclusion of common symbols, hyphens, and accented characters that are often found in legal names (e.g., 'Nagaratnam s/o Suppiah', 'Anya Taylor-Joy', 'Sergio Pérez'). Additionally, the app allows leading, trailing, and multiple consecutive spaces, which can result in inconsistent formatting.
+
+   This enhancement will:
+    - Loosen restrictions to allow symbols, hyphens, and accented characters, enabling the accurate entry of a wider range of legitimate names.
```

Comment:
Maybe can specify 'all other special characters'

Reply:
done

Reply:
for which para?